### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Nova281**
Made README compare to my forked repo master

### CODE REVIEW

**Comment by scottmc** [comment]
Please see comment on task.  I'd like to see the screenshot updated to have the tab, but have the background transparent.

**Comment by Nova281** [comment]
Ok

**Comment by scottmc** [comment]
See weather for example: https://github.com/HaikuArchives/Weather

**Comment by Nova281** [comment]
Got it!

**Comment by scottmc** [comment]
Looks good to me.  I'll defer to the other mentors for their review and if ok, let them merge it or request updates.

**Comment by Nova281** [comment]
Sounds good!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Features include:
+  * Opening a file dialog to load a document
+  * Print document
+  * View panel
+  * Zooming in or out
+  * Fitting the page length or width
+  * Shows thumbnail
+  * Find text
+  * Home panel giving access to other feature
+  * Options for help or to the website
```
Comment (humdingerb) [inline README.md]: A link to the project website isn't that much of a feature. How about just:
* Offline user documentation

Also, remove application/DocumentViewer_icon.png from the PR. We have the icon already in the root folder.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ 
+## Features include:
+  * Opening a file dialog to load a document
+  * Print document
+  * View panel
+  * Zooming in or out
+  * Fitting the page length or width
+  * Shows thumbnail
+  * Find text
+  * Home panel giving access to other feature
```
Comment (humdingerb) [inline README.md]: "...features"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+<img src="DocumentViewerScreenshot.png" height="360" width="480">
+ 
+## Features include:
+  * Opening a file dialog to load a document
+  * Print document
+  * View panel
+  * Zooming in or out
+  * Fitting the page length or width
+  * Shows thumbnail
```
Comment (humdingerb) [inline README.md]: "Show thumbnail"

File: README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+![DocumentViewer icon](DocumentViewer_icon.png)DocumentViewer
+=
+
+DocumentViewer is a viewer supporting PDF and DJVU files. 
+It provides many features, making it easy to open and view documents.<br>
+
+<img src="DocumentViewerScreenshot.png" height="360" width="480">
+ 
+## Features include:
+  * Opening a file dialog to load a document
```
Comment (humdingerb) [inline README.md]: That's not really a feature... remove.

**Pull request merged**